Sources/MapboxNavigation/Cache.swift

```diff
@@ -27,6 +27,52 @@ public protocol BimodalDataCache: BimodalCache {
     func data(forKey: String?) -> Data?
 }
 
+protocol BimodalURLCache {
+    func store(_ cachedResponse: CachedURLResponse, for request: URLRequest)
+    func data(for request: URLRequest) -> Data?
+    func clearCache()
+    func removeCache(for request: URLRequest)
+}
+
+/**
+ A general purpose URLCache used by `SpriteRepository` implementations.
+ */
+internal class URLDataCache: BimodalURLCache {
+    let capcity = 10 * 1024 * 1024
+    let diskCacheURL: URL = {
+        let fileManager = FileManager.default
+        let basePath = fileManager.urls(for: .documentDirectory, in: .userDomainMask).first!
```

Comment:
So do you think it's better that we use the `cachesDirectory` for the urlCache files? Because we could always re-download it and don't need the back-up for it.